docs/daemon/configuration/configuration.mdx

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 4
+sidebar_label: "Configuration"
+id: daemon-config
+title: Configuration
+description: How to configure the Kwil Daemon
+slug: /daemon/config/
+---
+
+Kwild application reads its config from the config.toml located in the home_dir/abci/config/config.toml. This include both the Kwildb and the chain configuration.
+
```

Comment:
You should probably mention both the config path as well as the root directory path that can be passed as flags here.